src/node.rs

```diff
@@ -469,7 +469,7 @@ impl Node {
     ///
     /// Only if we have a chain (meaning we are elders) we will process this API
     pub fn min_split_size(&self) -> Option<usize> {
-        self.chain().map(|chain| chain.min_split_size())
+        self.chain().map(|chain| chain.safe_section_size())
```

Comment:
Did you mean to update the function name to here min_split_size - > safe_section_size?

Reply:
Ah yes, the min _split _size was a wrapper around safe_section_size + SPLIT_BUFFER, so I removed that with the split buffer.  Later we can remove min_seciotn_size and just use safe_section_size or vice versa as we have 2 functions doing the same thing really.

Reply:
What I mean is that it is now a forwarding function called min_split_size that calls safe_section_size which is confusing, so this function should be renamed safe_section_size

Reply:
Ah yes, the RT min_section_size is forwarding, but will go away with the RT removal. We could change it here, but there is already a PR in play to remove RT.